src/coreclr/gc/unix/gcenv.unix.cpp

```diff
@@ -724,6 +728,11 @@ bool GCToOSInterface::VirtualCommit(void* address, size_t size, uint16_t node)
 {
     bool success = mprotect(address, size, PROT_WRITE | PROT_READ) == 0;
 
+#ifdef MADV_DODUMP
+    // Include committed memory in coredump.
+    madvise(address, size, MADV_DODUMP);
```

Comment:
should also only do this when success is true.

Reply:
yeah will fix, thought I had fixed all these yesterday.